.github/CODEOWNERS

```diff
@@ -83,6 +83,14 @@
 
 /eng/pipelines/**/*wasm* @radical
 
+# Linker codeowners
+/src/tools/linker/ @marek-safar
```

Comment:
I am looking at the list directory and file names in the PR. It is a mix of `linker` and `illink`. Should we take this as opportunity to work towards a single name?

Reply:
Given that all shipping artifacts are called `ILLink.*`, should the one name to standardize on be `ILLink`?

Reply:
> should the one name to standardize on be ILLink

Makes sense

Reply:
Yeah, sounds good to me, but I would limit the initial PR to just the directory name change.

Right now my highest priority is that we merge in a copy as close as possible to the original repo, then we can make refactoring commits. Doing both together just leads to an unreviewable diff.

Reply:
I would even say: I am fine leaving renaming the directory to a subsequent PR, as git handles move-only commits very well.

Reply:
I renamed the directory and all the usages of the name in https://github.com/dotnet/runtime/pull/78049